File: 20A40/cir-villalobos/completos/e34/Empleado.py
from datetime import datetime

class Empleado:
    nombre = "Empleado"
    salario = 0
    fechaNac = datetime.today()

    def __init__(self,nombreC, salarioC, fechaNacC):
        self.nombre = nombreC
        self.salario = salarioC
        self.fechaNac = fechaNacC
        
    def setNombre(self,nomb):
        if(not(len(nomb) > 0)):    return False
        self.nombre = nomb
        return True
    
    def setSalario(self,sala):
        if(not(sala > 0)):    return False
        self.salario = sala
        return True
    
    def setFechaNac(self, fech):
        try:
            self.fechaNac = fech
            return True
        except: return False

    def getNombre(self)  : return self.nombre
    def getSalario(self) : return self.salario
    def getFechaNac(self): return self.fechaNac
    
    def toString(self):
        concat = ("Nombre: " + self.nombre + "\tSalario: " + str(self.salario)
            + "\tFecha de nacimiento: " + str(self.fechaNac.today())[0:10])
        return concat

File: 20A40/cir-Enrique/Ejercicio 34/Empleado.java
public class Empleado {
    private String nombre;
    private double salario;
    private String fechaNac;
 
    Empleado(){}
    Empleado(String nombreC, double salarioC, String fechaNacC){
 
       this.nombre = nombreC;
       this.salario = salarioC;
       this.fechaNac = fechaNacC;
    }

    public boolean setNombre(String n){
       if (!n.isEmpty())
       {
          this.nombre = n;
          return false;
       } else {
          return true;
       }
    }

    public boolean setSalario(double s){
        if (s > 0)
            {
               this.salario = s;
               return true;
            } else {
               return false;
            }
    }

    public boolean setFechaNac(String f){
       try {
        this.fechaNac = f;
        return true;
       } catch (Exception e) {
        return false;
       }
    }


    public String getNombre(){
       return nombre;
    }
    
    public double getSalario(){
        return salario;
    }

    public String getFechaNac(){
        return fechaNac;
    }

    
    public String toString()
    {
        String concat = (this.nombre + "        " + this.salario + "       " + this.fechaNac);
       return concat;
    }
}
 